package.json

```diff
@@ -78,6 +78,7 @@
     "error-stack-parser": "^2.0.1",
     "expo": "^23.0.0",
     "express": "^4.16.2",
+    "filesize": "^3.5.11",
```

Comment:
This one is not added to `webpackAppConfig.js`

Reply:
It is really small and can be used on all platforms, so I did not add it.